x/bank/internal/types/expected_keepers.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+)
+
+// expected account keeper
+type AccountKeeper interface {
+	NewAccountWithAddress(ctx sdk.Context, addr sdk.AccAddress) auth.Account
```

Comment:
We should completely remove the dependency on `x/auth` by also having the `Account` interface here too.

Reply:
if I understand correctly, @alexanderbez is suggesting to use `authtypes.Account` instead of `auth.Account`

Reply:
That should work too!

Reply:
Is the goal to remove `x/auth` dependency all throughout `x/bank`? Then the `Account` interface would be necessary right, and not just the usage of `authtypes.Account` in `expected_keepers.go`?

Reply:
also if the goal is to remove `x/auth` dependency throughout `x/bank`, does this need to be applied to all the tests as well?

Reply:
The goal is to remove import module interdependency, yes. However, as @fedekunze suggested, we can safely use the types package from auth. I would not worry about the tests for now.